catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -109,6 +113,17 @@ public void init() {
           res.header("Content-Encoding", "gzip");
         });
 
+    get(
+        "/cql/transforms",
+        (req, res) -> {
+          List transformers =
+              queryResponseTransformers
+                  .stream()
+                  .map(serviceReference -> serviceReference.getProperty(TRANSFORMER_ID_PROPERTY))
```

Comment:
❓ Are we okay with the possible `null` this might return?

Reply:
Yes, that means no transformers available.

Reply:
That is a function of the JSON library. Most of them (gson, jackson, etc.) have the ability to enable/disable null output.

Reply:
Boon is configured to include null values: https://github.com/codice/ddf/blob/master/catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java#L61

Reply:
I'd rather not leave it up to the frontend to filter null values. Also if the `id == null` then when the user tries to export that result set the transformer won't be able to be found resulting in another error.